cmd/list.go

```diff
@@ -59,11 +57,12 @@ func init() {
 			// the expected argument in input is the container sas / or path of virtual directory in the container.
 			// verifying the location type
 			location := inferArgumentLocation(sourcePath)
-			if location != location.Blob() {
+			// Only support listing for Azure locations
+			if location != location.Blob() && location != location.File() && location != location.BlobFS() {
```

Comment:
>location [](start = 6, length = 8)

Just to double check, are there smoke tests covering the list command?

Reply:
No. We probably should have them.